core/src/main/java/org/mule/api/lifecycle/LifecycleUtils.java

```diff
@@ -22,11 +22,28 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LifecycleUtils.class);
 
+    /**
+     * if {@code object} implements the {@link Initialisable} interface,
```

Comment:
I would change the javadoc to something like "Invokes {@link Initialisable#initialise()} on a given object if it implements {@link Initialisable}"